### ISSUE TITLE
Graph Dataloader

### ISSUE THREAD

**Issue by DuanhaoranCC** [THUDM/GraphMAE#19]
```python
train_idx = torch.arange(len(graphs))
train_sampler = SubsetRandomSampler(train_idx)
    
train_loader = GraphDataLoader(graphs, sampler=train_sampler, collate_fn=collate_fn, batch_size=batch_size, pin_memory=True)
eval_loader = GraphDataLoader(graphs, collate_fn=collate_fn, batch_size=batch_size, shuffle=False)
```
I would like to ask the author why the two data loads are not shuffle. When LayerNorm and BanchNorm are used in the model, will there be any message leakage.

**Comment by chinmay5** [comment]
hi @DuanhaoranCC I have the same doubt.  Can you please tell me how there is no issue here?
